File: C11/백준/Bronze/18691. Pokemon Buddy/Pokemon Buddy.c
#include <stdio.h>

int main(void) {
	
	int T;
	scanf("%d", &T);
	
	for(int i = 0; i < T; i++) {
		int G, C, E;
		scanf("%d %d %d", &G, &C, &E);
		
		int kilometer = 0;
        
        switch(G) {
            case 1:
                kilometer = 1;
                break;
            case 2:
                kilometer = 3;
                break;
            case 3:
                kilometer = 5;
                break;
        }
		
		if (C >= E) {
			printf("%d\n", 0);
		} else {
			int diff = E - C;
			printf("%d\n", diff * kilometer);
		}
	}
	
	return 0;
}

File: Java/백준/Bronze/18691. Pokemon Buddy/Pokemon Buddy.java
import java.io.*;

public class Main {
    public static void main(String[] args) throws IOException {

        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));

        int T = Integer.parseInt(br.readLine()); //테스트 케이스 수
        for (int i = 0; i < T; i++) {
            String[] input = br.readLine().split(" ");
            int G = Integer.parseInt(input[0]); //그룹
            int C = Integer.parseInt(input[1]); //현재 사탕 수
            int E = Integer.parseInt(input[2]); //진화에 필요한 사탕 수

            int kilometer = switch (G) {
                case 1 -> 1;
                case 2 -> 3;
                case 3 -> 5;
                default -> 0;
            };

            if (C >= E) {
                System.out.println(0);
            } else {
                int diff = E - C;
                System.out.println(diff * kilometer);
            }
        }
    }
}